scripts/update-otel-deps.js

```diff
@@ -0,0 +1,348 @@
+#!/usr/bin/env node
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Update '@opentelemetry/*' deps in all workspaces.
+ *
+ * Usage:
+ *      # You should do a clean 'npm ci' before running this.
+ *      node scripts/update-otel-deps.js
+ *
+ * You can set the `DEBUG=1` envvar to get some debug output.
+ */
+
+const assert = require('assert');
+const fs = require('fs');
+const path = require('path');
+const {spawnSync} = require('child_process');
+
+const globSync = require('glob').sync; // TODO add top-level devDep on glob
+const minimatch = require('minimatch'); // TODO add top-level devDep on minimatch
+const semver = require('semver'); // TODO add top-level devDep on semver
```

Comment:
Note for reviewer: This is a little lame. It would be nice to add real top-level devDeps for these. However, I was rushing this PR to get a smaller (no package-lock.json changes) change for you to look at.